ingress/controllers/nginx/nginx/main.go

```diff
@@ -67,7 +67,9 @@ func NewManager(kubeClient *client.Client) *Manager {
 
 	ngx.sslDHParam = ngx.SearchDHParamFile(config.SSLDirectory)
 
-	ngx.loadTemplate()
+	if err := ngx.loadTemplate(); err != nil {
+		glog.Fatalf("invalid NGINX template: %v", err)
```

Comment:
> do you really want to fatal when a load fails? 

Only the first time is loaded (in case someone uses a custom template)

>  I think the answer is we should do whatever nginx would do if someone modified the nginx.conf to have a syntax error.

If the template cannot be loaded the pod will fail the health checks (crashloop) and the log doesn't help (check the output of 1448).